dotnet/samples/HomeAutomation/appsettings.Development.json

```diff
@@ -0,0 +1,5 @@
+{
+  "AzureOpenAiOptions": {
+    // "ApiKey": "" // Set value here, or using "dotnet user-secrets"
```

Comment:
Adjusting this file didn't work for me for some reason; I had to update the appsettings.json before the application picked up my configuration.

Reply:
I have addressed to root cause to this.  The repo was missing a file required for this file to be taken into consideration because it was it the Git ignore list.